pydoc/usage/random.rst

```diff
@@ -17,10 +17,26 @@ To create a ``Generator`` use the ``default_rng`` constructor
 Features
 ==========
 
+choice
+---------
+.. autofunction:: arkouda.random.Generator.choice
+
```

Comment:
So why are some functions added to the usage section of the docs and not others?  What is the intended difference between the usage and api sections?

Reply:
to be honest, this predates me. I don't really know. I think the api lists everything and usage is just meant for the most common stuff